### PR TITLE
Doc 10402  updates access control pages

### PR DESCRIPTION
**Pull request by franrob-projects**
- Removes duplicate page (Access Control Overview)

- Removes Access Control Overview from Nav

- Removes page title that's not duplicated in the nav

- Updates the Sync Function Page

- Updates Users page

- Updates Roles page

- Updates Channel page

- Updates Import Filters page

### CODE REVIEW

File: modules/ROOT/pages/_partials/concepts/channels.adoc [hunk 1]
```diff
@@ -7,19 +7,17 @@
 
 // tag::full[]
 // tag::summary[]
-Sync Gateway uses _Channels_ to make it easy to share a database's documents across a large user base whilst retaining effective access control.
+Sync Gateway uses _Channels_ defined at collection level to make it easy to share a database's documents across a large user base whilst retaining effective access control.
```
Comment (dinaabennett) [inline modules/ROOT/pages/_partials/concepts/channels.adoc:10]: while :-)

File: modules/ROOT/pages/_partials/concepts/channels.adoc [hunk 2]
```diff
@@ -33,20 +31,9 @@ Sync Gateway provides two special channels and a channel wildcard character.
...
-
-image::channels-example-all.png["Access Control using Channels",400]
-
-In this case, each user is assigned to a private channel with a subset of documents that only the user can access and a shared channel that contains common documents.
-The users in this example could be replaced by roles.
-
-An example of this model could be a retail chain, where every store corresponds to a Sync Gateway User.
-Each store has a store specific channel that contains documents that are store specific like pricing, promotions, inventory etc.
-In addition, all the stores may need to have access to a common set of documents like retail locations, product catalog that are stored in a common shared channel.
+Imagine a database containing a collection of customer data. Within this collection, channels can be established to provide varying customer data access levels.
```
Comment (dinaabennett) [inline modules/ROOT/pages/_partials/concepts/channels.adoc:34]: You have a database containing a collection of customer data. Within this collection, channels can be established to provide varying customer data access levels. 
One channel can provide access to general customer information, while another can allow access to sensitive customer information. 
Users can then be granted access to one or both channels based on their access permissions.

File: modules/ROOT/pages/_partials/concepts/roles.adoc [hunk 3]
```diff
@@ -11,7 +11,9 @@ They enable the grouping together of {users--xref} with similar characteristics,
 // end::summary[]
 // tag::body[]
 
-Roles are granted access to channels. Any user assigned a role can access any channels (and documents within those channels) the role has been granted access to.
+A Role and a user assigned to a Role is granted access to a channel, a part of collections.
```
Comment (dinaabennett) [inline modules/ROOT/pages/_partials/concepts/roles.adoc:14]: You grant a role access to a channel which is part of a collection. A user assigned to that role is granted access to that channel. 
The user can then access all documents assigned to that channel. 
When granting user access to admin channels statically, the user must specify the collection and corresponding channel.
[does that make sense?]

File: modules/ROOT/pages/_partials/concepts/sync-function.adoc [hunk 4]
```diff
@@ -116,7 +133,8 @@ Your implementation of the sync function can omit the `oldDoc` parameter if you
 
 This argument references the user defined XATTR that you can use to hold access grant data.
 
-The referenced object can include items such as channels or roles. So instead of embedding channel information directly within the document body, users can specify the user-defined XATTR associated with the document -- see {access-control-how-use-xattrs-for-access-grants--xref}.
+The referenced object can include items such as channels or roles. 
+So instead of embedding channel information directly within the document body, users can specify the user-defined XATTR associated with the document -- see {access-control-how-use-xattrs-for-access-grants--xref}.
```
Comment (dinaabennett) [inline modules/ROOT/pages/_partials/concepts/sync-function.adoc:142]: Remove "so"

File: modules/ROOT/pages/_partials/concepts/users.adoc [hunk 5]
```diff
@@ -18,7 +18,8 @@ As an entity a _user_ comprises a name, password, list of {roles--xref} and a li
 Sync Gateway users and roles have no relationship to <<lbl-rbac-users,Couchbase Server's _RBAC (Role-based Access Control) users_>>.
 They are created and operate solely within the {sgw} ecosphere to govern access to replication data and to the Public API.
 
-Users granted access to a {channel--xref} can access all documents assigned to that channel.
+Granting access to a channel in Couchbase Sync Gateway allows users to access all documents assigned to that channel, which are part of specific collections. 
+If granting users access to admin channels statically, it is necessary to specify both the corresponding collection and channel.
```
Comment (dinaabennett) [inline modules/ROOT/pages/_partials/concepts/users.adoc:22]: You must specify the corresponding collection and channel when granting users access to admin channels statically.

**Review by dinaabennett — APPROVED** [review]
I left some suggestions :-)

File: modules/ROOT/pages/_partials/concepts/sync-function.adoc [hunk 6]
```diff
@@ -39,53 +39,70 @@ Any validation checks will probably fail because of the missing properties, so b
...
-
-Version 3.x::
-+
 --
 
 [source, javascript]
 ----
-function (doc, oldDoc, meta) { // <.>
-   channel(doc.channels); // <.>
+Sync (doc, oldDoc, meta) {
```
Comment (osfameron) [inline modules/ROOT/pages/_partials/concepts/sync-function.adoc]: careful. This is Javascript: `function` was correct.
(Priya was handwaving in her comment on JIRA. The syntax of the sync functions hasn't changed, that we know of.)

File: modules/ROOT/pages/_partials/concepts/sync-function.adoc [hunk 7]
```diff
@@ -39,53 +39,70 @@ Any validation checks will probably fail because of the missing properties, so b
...
-[#ex-prototype]
-.Prototype Sync Function
+Couchbase Sync Gateway defines a Sync Function at the `collection` level. 
+Defining at this level helps simplify data management and improve data reliability. 
+Each collection in the system allows for only one Sync Function, which enables the specification of Access Control rules.
+
+.Default Sync Function
 ====
 
-[{tabs}]
```
Comment (osfameron) [inline modules/ROOT/pages/_partials/concepts/sync-function.adoc:48]: Just want to check why you got rid of the tabs here?
Also, any oddities from the successive `====` and `=====` blocks now?

File: modules/ROOT/pages/_partials/concepts/sync-function.adoc [hunk 8]
```diff
@@ -39,53 +39,70 @@ Any validation checks will probably fail because of the missing properties, so b
...
+
+=====
 --
 
 [source, javascript]
 ----
-function (doc, oldDoc) {
-   channel(doc.channels); // <.>
-}
+Sync (doc, oldDoc, meta) {
```
Comment (osfameron) [inline modules/ROOT/pages/_partials/concepts/sync-function.adoc]: Again, `function` was correct

File: modules/ROOT/pages/_partials/concepts/sync-function.adoc [hunk 9]
```diff
@@ -39,53 +39,70 @@ Any validation checks will probably fail because of the missing properties, so b
...
+
+.Upgraded Default Sync Function
+====
 
+=====
 --
 
+[source, javascript]
+----
+Sync (doc, oldDoc, meta) {
```
Comment (osfameron) [inline modules/ROOT/pages/_partials/concepts/sync-function.adoc]: `function`

**Review by osfameron — CHANGES_REQUESTED** [review]
Good work, thanks! 
Just one important change - `function` rather than Sync.

**Review by osfameron — APPROVED** [review]
fab, thanks 👍

**Pull request merged**